backend/internal/application/service/applicationservice.go

```diff
@@ -39,9 +39,9 @@ import (
 
 // ApplicationServiceInterface defines the interface for the application service.
 type ApplicationServiceInterface interface {
-	GetOAuthApplication(clientID string) (*model.OAuthAppConfigProcessed, *serviceerror.ServiceError)
 	CreateApplication(app *model.ApplicationDTO) (*model.ApplicationDTO, *serviceerror.ServiceError)
 	GetApplicationList() (*model.ApplicationListResponse, *serviceerror.ServiceError)
+	GetOAuthApplication(clientID string) (*model.OAuthAppConfigProcessed, *serviceerror.ServiceError)
 	GetApplicationByID(appID string) (*model.ApplicationProcessedDTO, *serviceerror.ServiceError)
 	GetApplicationByName(appName string) (*model.ApplicationProcessedDTO, *serviceerror.ServiceError)
```

Comment:
Do we need this `GetApplicationByName()`?

Reply:
This is used in create application method to check whether a application exists with the same name before creation

Reply:
Removed service layer method with ec15851905fe6f88d0c1a113abd60b0f2eb30166